scripts/README.md

```diff
@@ -28,18 +28,9 @@ openshift_hosted_registry_storage_volume_size=10Gi
 openshift_hosted_registry_storage_host=n.n.n.n
 ```
 
-## Installation Scripts
-
-Retrieve the [installation scripts from our documentation repository](https://github.com/kabanero-io/kabanero-foundation/tree/master/scripts)
-
 ## Installation
 
-Review the installation script for any optional components that you wish to enable.  Components can be enabled by passing the appropriate environment variable to the script, or by editing the script.
-
-As a `cluster-admin`, execute the installation script, replacing `my.openshift.master.default.subdomain` with your subdomain:
-```
-openshift_master_default_subdomain=<my.openshift.master.default.subdomain> ./install-kabanero-foundation.sh
-```
+View [Installing Kabanero Foundation](https://kabanero.io/docs/ref/general/installing-kabanero-foundation.html) on Kabanero.io
```

Comment:
Well, I think this is a great idea, to just have one version, but my concern is that the kabanero-foundation repository is versioned.  If the readme points them to kabanero.io, the instructions that they get might not reflect the version that they are using.  

Maybe I am over-thinking this... we don't provide instructions today to get the version of the install script at a particular version of Kabanero.

Reply:
@kaczyns we will soon have versioned docs on the site, maybe within a week. This link will always go to the latest doc, but the user will soon see a drop down to get a specific versioned install doc, which hopefully should be enough to address this problem